tests/utest/test_listing_rules.py

```diff
@@ -135,6 +135,39 @@ def test_list_disabled_rule(self, robocop_pre_load, msg_0101, msg_disabled_for_4
             "Visit https://robocop.readthedocs.io/en/stable/rules.html page for detailed documentation.\n"
         )
 
+    def test_list_filter_enabled(self, robocop_pre_load, msg_0101, msg_0102_0204, capsys):
+        robocop_pre_load.config.list = "ENABLED"
+        init_empty_checker(robocop_pre_load, msg_0101)
+        init_empty_checker(robocop_pre_load, msg_0102_0204, exclude=True)
+        with pytest.raises(SystemExit):
+            robocop_pre_load.list_checkers()
+        out, _ = capsys.readouterr()
+        assert (
+            out == "Rule - 0101 [W]: some-message: Some description (enabled)\n\n"
+            "Altogether 1 rule with following severity:\n"
+            "    0 error rules,\n"
+            "    1 warning rule,\n"
+            "    0 info rules.\n\n"
+            "Visit https://robocop.readthedocs.io/en/stable/rules.html page for detailed documentation.\n"
+        )
+
+    def test_list_filter_disabled(self, robocop_pre_load, msg_0101, msg_0102_0204, capsys):
+        robocop_pre_load.config.list = "DISABLED"
+        init_empty_checker(robocop_pre_load, msg_0101)
+        init_empty_checker(robocop_pre_load, msg_0102_0204, exclude=True)
+        with pytest.raises(SystemExit):
+            robocop_pre_load.list_checkers()
+        out, _ = capsys.readouterr()
+        assert (
+            out == "Rule - 0102 [E]: other-message: this is description (disabled)\n",
+            "Rule - 0204 [I]: another-message: Message with meaning 4 (disabled)\n",
+            "Altogether 1 rule with following severity:\n"
+            "    0 error rules,\n"
+            "    2 warning rule,\n"
```

Comment:
I think this will have plural form:
```suggestion
            "    2 warning rules,\n"
```

Reply:
You're right and it should be catched by test itself - due to string formatting issues it only compare with line. I fixed the string formatting and string itself so now the assertion is correct.